Dfe.PrepareConversions/Dfe.PrepareConversions/Pages/TaskList/SchoolBudgetInformation/UpdateSchoolBudgetInformation.cshtml

```diff
@@ -58,4 +58,21 @@
          </button>
       </form>
    </div>
-</div>
\ No newline at end of file
+
+   <div class="govuk-grid-column-one-third">
+      <aside class="app-related-items" role="complementary">
+         <h2 class="govuk-heading-s" id="subsection-title">
+            Useful information
+         </h2>
+         <nav role="navigation" aria-labelledby="subsection-title">
+            <ul class="govuk-list govuk-!-font-size-16">
+               <li>
+                  <p class="govuk-body-s">
+                     If you <a asp-page="@Links.AnnexB.Index.Page" asp-route-id="@Model.Project.Id">attach an Annex B form</a> to this project, it will appear here for reference.
```

Comment:
Is Annex B only for Sponsored? If so we could do the flag on this

Reply:
That's a good point! 🦾

Reply:
Done!